apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -0,0 +1,156 @@
+import { ChevronDownIcon } from "@radix-ui/react-icons";
+import { Suspense } from "react";
+
+import Table from "#/components/Table";
+import votingGauges from "#/data/voting-gauges.json";
+import { Pool } from "#/lib/balancer/gauges";
+import { formatNumber } from "#/utils/formatNumber";
+
+import BalancerAPI from "../../(utils)/balancerAPI";
+import calculateRoundAPR from "../../(utils)/calculateRoundAPR";
+import { getBALPriceByRound } from "../../(utils)/getBALPriceByRound";
+import { getPoolRelativeWeight } from "../../(utils)/getRelativeWeight";
+import { Round } from "../../(utils)/rounds";
+import ClientSideBodyRow from "./ClientSideBodyRow";
+
+export default async function PoolListTable({ roundId }: { roundId: string }) {
+  return (
+    <div className="overflow-hidden rounded-lg w-full text-white">
+      <Table color="blue" variant="solid" shade="dark">
+        <Table.HeaderRow>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>TVL</Table.HeaderCell>
+          <Table.HeaderCell> Votes % </Table.HeaderCell>
+          <Table.HeaderCell>APR</Table.HeaderCell>
+        </Table.HeaderRow>
+        <Table.Body>
+          {votingGauges.slice(0, 10).map((gauge) => (
```

Comment:
we should be able to order this table by the columns -- can you add a task for it?

Reply:
yup! [Here](https://linear.app/bleu-llc/issue/BAL-651/order-pool-list-by-the-columns)